electrum/submarine_swaps.py

```diff
@@ -178,6 +176,30 @@ def start_network(self, *, network: 'Network', lnwatcher: 'LNWalletWatcher'):
                 continue
             self.add_lnwatcher_callback(swap)
 
+    async def pay_pending_invoices(self):
+        # for server
+        self.invoices_to_pay = set()
+        while True:
+            await asyncio.sleep(1)
+            for key in list(self.invoices_to_pay):
+                swap = self.swaps.get(key)
+                if not swap:
+                    continue
+                invoice = self.wallet.get_invoice(key)
+                if not invoice:
+                    continue
+                current_height = self.network.get_local_height()
+                delta = current_height - swap.locktime
+                if delta > - 5:
```

Comment:
The same security parameter is 60 for the client. Please use the same value and factor them into a global named value.
5 blocks' time is not enough to get a transaction mined.

Reply:
Arguably, even 60 is low.